### DISCUSSION TITLE
Click on Show dependencies for a measure, but nothing happens

### DISCUSSION THREAD

**Discussion by InsightfulAnalytics** [TabularEditor/TabularEditor3#757] [🙋♀️TE3 Q&A] [answered]
Similar to the issue here:

[https://github.com/TabularEditor/TabularEditor/issues/208](url)

But this happens with any measure.  I click on show dependencies and nothing happens.

**Comment by otykier** [comment, marked as answer]
Hi. Most likely, the dependency view window is showing off-screen, see #652. Could you try to reset your window layout, **Window > Default Layout**?

**Reply by InsightfulAnalytics** [reply]
that worked a treat thank you.  Not sure where the other window was going, I suspected this could be happening so I minimised all other programs but still couldn't see anything, odd, it was probably being sent to a non-existent location, but either way all good now.
